opencti-platform/opencti-front/src/private/components/settings/decay/DecayRulesLine.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { FunctionComponent } from 'react';
+import { graphql, useFragment } from 'react-relay';
+import { Link } from 'react-router-dom';
+import { KeyboardArrowRightOutlined } from '@mui/icons-material';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import makeStyles from '@mui/styles/makeStyles';
+import Skeleton from '@mui/material/Skeleton';
+import ItemIcon from '../../../../components/ItemIcon';
+import { DataColumns } from '../../../../components/list_lines';
+import type { Theme } from '../../../../components/Theme';
+import { DecayRulesLine_node$key } from './__generated__/DecayRulesLine_node.graphql';
+
+const useStyles = makeStyles<Theme>((theme) => ({
```

Comment:
`makeStyles` is deprecated. Try to put style directly in JSX using `style={{...}}` for html elements or `sx={{...}}` for MUI elements